src/include/ggponet.h

```diff
@@ -8,6 +8,21 @@
 #ifndef _GGPONET_H_
 #define _GGPONET_H_
 
+#if defined(_MSC_VER)
```

Comment:
This pull request did something similar which will conflict with a lot of these changes.  Could you re-pull `dev/crossplatform` into your fork and restage?